dockers/broad/genomes_on_the_cloud/build_docker.sh

```diff
@@ -16,7 +16,6 @@ GATK35_VERSION="3.5-0-g36282e4"
 GATK4_VERSION="4.1.8.0"
```

Comment:
Not on this line, but please change DOCKER_VERSION (line 10) to 2.4.8 (minor version change - should be functionally the same…)